roles/process_kvm_nodes/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+vm_params:
+  master:
+    memory: "{{ vm_spec_master_memory | default(16384) }}"
+    vcpu: "{{ vm_spec_master_vcpu | default(6) }}"
+    disk_size: "{{ vm_spec_master_disk | default(30) }}"
```

Comment:
It's a little bit inconsistent to have one thing named `vm_params.master.disk_size` and the other `vm_spec_master_disk`. 
Maybe we could change `vm_spec_master_disk` to `vm_spec_master_disk_size`?